speech_recognition/__init__.py

```diff
@@ -1390,7 +1390,25 @@ def recognize_tensorflow(self, audio_data, tensor_graph='tensorflow-data/conv_ac
             for node_id in top_k:
                 human_string = self.tflabels[node_id]
                 return human_string
-
+            
+    def recognize_vosk(self, audio_data, language='en'):
+        from vosk import Model, KaldiRecognizer
+        
+        assert isinstance(audio_data, AudioData), "Data must be audio data"
+        assert isinstance(language, str), "Language data"
+        
+        if not hasattr(self, 'vosk_model'):
+            if not os.path.exists("model"):
+                return "Please download the model from https://github.com/alphacep/vosk-api/blob/master/doc/models.md and unpack as 'model' in the current folder."
+                exit (1)
+            self.vosk_model = Model()
```

Comment:
should be Model(language) or at least Model("model")

Reply:
Done